zerver/worker/queue_processors.py

```diff
@@ -427,6 +430,18 @@ def consume(self, event):
 class OutgoingWebhookWorker(QueueProcessingWorker):
     def consume(self, event):
         # type: (Mapping[str, Any]) -> None
-
-        # TODO: Implement this worker.
-        pass
+        message = event['message']
+        services = get_bot_services(event['user_profile_id'])
+        rest_operation = {'method': 'POST',
+                          'relative_url_path': '',
+                          'request_kwargs': {},
+                          'base_url': ''}
+        dup_event = dict()
```

Comment:
`dup_event = cast(Dict[str, Any], event)` is working, this should solve our need to substitute the loop right?